### DISCUSSION TITLE
Support for axial chirality in atropisomeric substituted biaryl compounds

### DISCUSSION THREAD

**Discussion by drassokhin** [rdkit/rdkit#5584] [General] [answered]
Does the current version of RDKit support axial chirality in atropisomeric substituted biaryl compounds (detection and M/P label assignment)?

**Comment by greglandrum** [comment, marked as answer]
@drassokhin It does not.
Providing support for representing atropisomers and doing the M/P label assignment is on my "I hope ToDo" list for the 2023.03 release.

**Reply by drassokhin** [reply]
Greg, thanks for your quick response. A somewhat related question: are you aware of any efforts towards further enhancing SMILES to make the encoding of the atropisomeric axial chirality info possible? It does not seem that even the latest extended ChemAxon SMILES can do that.

**Reply by greglandrum** [reply]
I think the only option currently available would be to use either the bond wedging information available in CXSMILES with coordinates (“wU:” and “wD:”) or to use SGroup data on the bond with hindered rotation. These are also the ways that I know of to encode the info in Mol blocks

**Reply by GLPG-GT** [reply]
Just a note: Biovia uses wedge bonds on sp2 atoms to encode what they call 'Stereochemistry of Allenes and Biaryls'.
It is quite readable 'by eye', as opposed to Sgroup data, which would need to be interpreted.
From the Biovia 2020 reference guide:
-for allenes:
![image](https://user-images.githubusercontent.com/96192927/232727089-0ba46bb5-d6b7-4b3d-8164-4c17aba454cc.png)
-for biaryls:
![image](https://user-images.githubusercontent.com/96192927/232727212-5819cd4d-b434-446a-a47c-10e9829080be.png)
(although here I think they should have set the single bond to wedge, not the double one)

**Comment by YojanaGadiya** [comment]
@greglandrum is this function already implemented in rdkit latest version?

**Reply by greglandrum** [reply]
Not yet. We might have a first iteration available in the 2023.09 release, but that's not certain

**Comment by katrze89** [comment]
@greglandrum  what is the status of this function? And where I can check it?

**Reply by greglandrum** [reply]
We have merged a PR from @tadhurst-cdd with a first implementation of support for atropisomeric bonds. This will be available in the 2024.03.1 release, scheduled for the end of March.

Here's a link to the documentation of what is coming:
https://github.com/rdkit/rdkit/blob/master/Docs/Book/RDKit_Book.rst#L2166

**Reply by katrze89** [reply]
thank you.
